subprojects/gradle/cd/src/test/kotlin/com/avito/ci/RunTask.kt

```diff
@@ -0,0 +1,38 @@
+package com.avito.ci
+
+import com.avito.test.gradle.TestResult
+import com.avito.test.gradle.ciRun
+import java.io.File
+
+internal fun runTask(
+    projectDir: File,
+    vararg args: String,
+    branch: String = "develop",
+    dryRun: Boolean = true,
+    expectedFailure: Boolean = false
+): TestResult {
+    return ciRun(
```

Comment:
It will make harder realize what properties are useful
I consider some of them could be deleted

Reply:
i'll write an issue with my proposal to make our gradle test library more configurable